### PR TITLE
[move-examples] Move to using _ placeholder notation in Move.toml

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
The build system doesn't particularly like it when you provide an additional address for one that's hardcoded in the Move.toml.  This changes to the placeholder notation.

### Test Plan
CI should still pass if there's any automation that actually needed the fake addresses

### CODE REVIEW

**Review by 0xchloe — APPROVED** [review]

File: aptos-move/move-examples/resource_account/Move.toml [hunk 1]
```diff
@@ -8,4 +8,4 @@ AptosFramework = { local = "../../framework/aptos-framework" }
 
 [addresses]
 # change it to `source_addr = "_" when calling `create-resource-account-and-publish-package` from the cli
```
Comment (movekevin) [inline aptos-move/move-examples/resource_account/Move.toml:10]: Can we delete this comment

File: aptos-move/move-examples/mint_nft/4-Getting-Production-Ready/Move.toml [hunk 2]
```diff
@@ -9,4 +9,4 @@ AptosToken = { local = "../../../framework/aptos-token" }
 [addresses]
 aptos_framework = "0x1"
 # replace the admin_addr with the actual admin address we created using CLI
```
Comment (movekevin) [inline aptos-move/move-examples/mint_nft/4-Getting-Production-Ready/Move.toml:11]: Can we delete this comment

**Review by movekevin — APPROVED** [review]

**Comment by lightmark** [comment]
I am curious "The build system doesn't particularly like it" in what way?

**Comment by movekevin** [comment]
> I am curious "The build system doesn't particularly like it" in what way?

Compilation returns a weird error:
`{
  "Error": "Unexpected error: Unable to resolve packages for package 'test_packages'"
}`

**Comment by gregnazario** [comment]
We can't use this, looks like we need changes to the unit test framework.  I'm closing this for now.

**Pull request closed without merging**